plugins/modules/cloud/linode/linode.py

```diff
@@ -589,12 +592,12 @@ def main():
             alert_cpu_threshold=dict(type='int'),
             alert_diskio_enabled=dict(type='bool'),
             alert_diskio_threshold=dict(type='int'),
-            backupsenabled=dict(type='int'),
+            backupsenabled=dict(type='int', removed_in_version='5.0.0', removed_from_collection='community.general'),
```

Comment:
@aminvakil as far as I know, when I add this ^ to the parameter, it will already go to the docs. @felixfontein please correct me if I am wrong here.

Reply:
It won't go to the docs, it will only inform users when using the module parameter.